server/api.js

```diff
@@ -2,11 +2,40 @@ import { Router } from "express";
 
 import logger from "./utils/logger";
 
+import db from "./db";
+
 const router = Router();
 
 router.get("/", (_, res) => {
 	logger.debug("Welcoming everyone...");
 	res.json({ message: "Hello, world!" });
 });
 
+router.get("/energizers", async (_, res) => {
+	logger.debug("Welcoming everyone...");
+	const query = "select * from energizers";
```

Comment:
So I would say it's not very good practice to return every column from a database directly to the client, for now it would work but as you add columns they would get automatically included here which might not be intended.

Also minor point by SQL reserved words (SELECT, FROM) are generally always written uppercase.

Reply:
Thank you, good points, I have updated the SQL reserved words to uppercase I will try sticking to that convention, regarding the return of every column its only what I have been exposed to during the course, what do you suggest would be the better way to return from database